### DISCUSSION TITLE
/timer-trigger-azure-function-run-unforeseen-times-/

### DISCUSSION THREAD

**Discussion by comment-migrator** [estruyf/eliostruyf-comments#428] [Comments]
Why does your timer trigger Azure Function run at unforeseen times?
        
[https://www.eliostruyf.com/timer-trigger-azure-function-run-unforeseen-times-/](https://www.eliostruyf.com/timer-trigger-azure-function-run-unforeseen-times-/)

**Comment by cervezerg** [comment]
Hi, thank you for your post. Indeed this help me a lot, I was confused about why my functionApp in Azure was triggering a lot of times, and was because of this setting.
Best regards!
